packages/web/src/components/range/RatingsFilter.js

```diff
@@ -121,12 +121,12 @@ class RatingsFilter extends Component {
 		return value ? [value.start, value.end] : null;
 	};
 
-	static defaultQuery = (value, props, includeNullValues = false) => {
+	static defaultQuery = (value, props, includeUnrated = false) => {
 		let query = null;
 		if (value) {
 			query = getRangeQueryWithNullValues(value, {
 				dataField: props.dataField,
-				includeNullValues: props.includeNullValues || includeNullValues,
+				includeNullValues: props.includeNullValues || includeUnrated,
```

Comment:
@ShahAnuj2610 I'm just curious to know that how these(`includeNullValues` & `includeUnrated`) props are related to each other?

Reply:
the query executed for both these parameters are same, only `includeUnrated` is applied at the item level, whereas `includeNullValues` is applied for every item in the data
`
<RatingsFilter
					componentId="RatingsSensor"
					dataField="average_rating_rounded"
					title="RatingsFilter"
					data={[
						{ start: 4, end: 5, label: '4 stars and up', includeUnrated: true },
						{ start: 3, end: 5, label: '3 stars and up' },
						{ start: 2, end: 5, label: '2 stars and up' },
						{ start: 1, end: 5, label: '> 1 stars' },
					]}
				/>
`
In the above if `includeNullValues=false`, then null results will only be included for the first data item